x/pep/module/module.go

```diff
@@ -277,6 +277,24 @@ func (am AppModule) BeginBlock(cctx context.Context) error {
 			continue
 		}
 
+		// execute registered contracts
+		contracts, found := am.keeper.GetContractEntriesByID(ctx, entry.Identity)
+		if found {
+			if len(contracts.Contracts) != 0 {
+				for _, contract := range contracts.Contracts {
```

Comment:
I would add some check to the fields in `contract` before `ExecuteContract()` just to make sure is not empty
and I think we can: `if found && len(contracts.Contracts) != 0 {}` instead 2 if

Reply:
I mean just to make sure `entry.Identity`, `entry.Pubkey` and `entry.AggrKeyshare` are not empty before executing ?